*
* Visual Studio Code Server
*
* By using the software, you agree to
* the Visual Studio Code Server License Terms (https://aka.ms/vscode-server-license) and
* the Microsoft Privacy Statement (https://privacy.microsoft.com/en-US/privacystatement).
*
Server bound to 127.0.0.1:46373 (IPv4)
Extension host agent listening on 46373

[11:34:11] Extension host agent started.
[11:34:12] [127.0.0.1][003a3abf][ManagementConnection] New connection established.
[11:34:12] [127.0.0.1][b7ec995e][ExtensionHostConnection] New connection established.
[11:34:12] [127.0.0.1][b7ec995e][ExtensionHostConnection] <20678> Launched Extension Host Process.
[11:34:16] [127.0.0.1][b7ec995e][ExtensionHostConnection] <20678> Extension Host Process exited with code: 0, signal: null.
Last EH closed, waiting before shutting down
[11:34:16] Last EH closed, waiting before shutting down
Got delay-shutdown request while in shutdown timeout, delaying
[11:34:21] Got delay-shutdown request while in shutdown timeout, delaying
Cancelling previous shutdown timeout
[11:34:21] Cancelling previous shutdown timeout
[11:34:21] [127.0.0.1][f91a469a][ManagementConnection] New connection established.
[11:34:21] [127.0.0.1][5c4f6ca2][ExtensionHostConnection] New connection established.
[11:34:21] [127.0.0.1][5c4f6ca2][ExtensionHostConnection] <20861> Launched Extension Host Process.
[11:34:23] [127.0.0.1][003a3abf][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[11:34:23] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
rejected promise not handled within 1 second: ErrorNoTelemetry: Could not find pty on pty host
stack trace: ErrorNoTelemetry: Could not find pty on pty host,    at d._throwIfNoPty (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103),    at d.updateTitle (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349),    at Object.call (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293),    at onPromise (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880),    at onRawMessage (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268),    at /home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556,    at h.invoke (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:11:145),    at z.deliver (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275),    at M.fire (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853),    at process.W (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)
[11:34:24] [ErrorNoTelemetry: Could not find pty on pty host,    at d._throwIfNoPty (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:24:29103),    at d.updateTitle (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:24:25349),    at Object.call (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:16:8293),    at onPromise (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:15:4880),    at onRawMessage (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:15:4268),    at /home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:15:3556,    at h.invoke (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:11:145),    at z.deliver (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:11:2275),    at M.fire (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:11:1853),    at process.W (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/platform/terminal/node/ptyHostMain.js:9:20668),    at process.emit (node:events:390:28),    at emit (node:internal/child_process:917:12),    at processTicksAndRejections (node:internal/process/task_queues:84:21)]
New EH opened, aborting shutdown
[11:39:21] New EH opened, aborting shutdown
[11:52:54] [127.0.0.1][f91a469a][ManagementConnection] The client has disconnected, will wait for reconnection 3h before disposing...
[11:52:54] Error: Unexpected SIGPIPE
    at process.<anonymous> (/home/peci2022/.vscode-server/bin/b06ae3b2d2dbfe28bca3134cc6be65935cdfea6a/out/vs/server/node/server.main.js:145:7933)
    at process.emit (node:events:390:28)
[14:34:31] [127.0.0.1][003a3abf][ManagementConnection] The reconnection grace time of 3h has expired, so the connection will be disposed.
[14:53:03] [127.0.0.1][f91a469a][ManagementConnection] The reconnection grace time of 3h has expired, so the connection will be disposed.
[14:53:04] [127.0.0.1][5c4f6ca2][ExtensionHostConnection] <20861> Extension Host Process exited with code: 0, signal: null.
Last EH closed, waiting before shutting down
[14:53:04] Last EH closed, waiting before shutting down
Last EH closed, shutting down
[14:58:04] Last EH closed, shutting down
